src/Components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { TodoItem } from '../TodoItem';
+import { TodosContext } from '../../Context/TodosContext';
+import { Status } from '../Status/Status';
+
+export const TodoList: React.FC = () => {
+  const { todos, filter } = useContext(TodosContext);
+
+  const filteredTodos = () => {
+    switch (filter) {
+      case Status.ALL:
+        return todos;
+      case Status.ACTIVE:
+        return todos.filter((todo) => !todo.completed);
+      case Status.COMPLETED:
+        return todos.filter((todo) => todo.completed);
+      default:
+        return todos;
```

Comment:
```suggestion
      Status.ALL:
      default:
        return todos;
```
you may join cases if you do the same action for them